internal/envoy/envoy.go

```diff
@@ -88,7 +82,7 @@ func (srv *Server) Run(ctx context.Context) error {
 
 	srv.cmd = exec.CommandContext(ctx, envoyPath,
 		"-c", configFileName,
-		"--log-level", srv.logLevel,
+		"--log-level", "trace",
```

Comment:
Will this be updated by config/log.go or is that just for the main log level? Is the envoy log level now hardcoded to trace?

Reply:
this is so all the logs are available in the handler. Whether they are displayed or not is a result of the zerolog log level.

Reply:
@calebdoxsey perfect

Reply:
I recall we had some performance problems when we initial switched over to envoy.  https://github.com/pomerium/pomerium/pull/779 was pulling in a default that, I believe, set us to trace and slowed down performance significantly.  I might be misremembering but we should check this.